app/src/main/java/com/moviefiend/torian/moviefiend/MovieListFragment.java

```diff
@@ -0,0 +1,60 @@
+package com.moviefiend.torian.moviefiend;
+
+import android.app.Fragment;
+import android.app.LoaderManager;
+import android.content.Loader;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.moviefiend.torian.moviefiend.network.NowPlayingResponse;
+
+import java.util.ArrayList;
+
+public class MovieListFragment extends Fragment implements LoaderManager.LoaderCallbacks<ArrayList<NowPlayingResponse.MovieInfo>> {
```

Comment:
Any special reason why you decide to make it a fragment (besides reusing it in step 6 :))?

That's fine for the training, but one thing to consider is that now you have an activity and its layout that are completely empty.
So in my opinion you're not gaining much by having both an activity and a fragment.